animatedModal.js

```diff
@@ -29,23 +30,57 @@
             animationDuration:'.6s', 
             overflow:'auto', 
             // Callbacks
-            beforeOpen: function() {},           
+            beforeOpen: function(event, callback) {
+                callback();
+            },           
             afterOpen: function() {}, 
             beforeClose: function() {}, 
             afterClose: function() {}
  
+        }, options);
+
+        me.openModal = function(){
```

Comment:
extracted this out so it can be called manually
i.e. 
this.modalWindow = $('#foo').animatedModal({modalTarget: 'tipsSeenModal', animatedIn: 'slideInRight', animatedOut: 'bounceOutRight'});

this.modalWindow.openModal();
this.modalWindow.closeModal();